cli50.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+
+import argparse
+import distutils.spawn
+import os
+import pkg_resources
+import re
+import signal
+import subprocess
+import sys
+
+
+# Require python 2.7
+if sys.version_info < (2, 7):
+    sys.exit("CS50 CLI requires Python 2.7 or higher")
+if sys.version_info < (3, 0):
+    input = raw_input
+
+# Get distribution
+try:
+    __version__ = pkg_resources.get_distribution(module.__name__).version
+except:
+    __version__ = "UNKNOWN"
+
+
+def main():
+
+    # Listen for ctrl-c
+    signal.signal(signal.SIGINT, handler)
+
+    # Parse command-line arguments
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-f", "--fast", action="store_true", help="skip autoupdate")
+    parser.add_argument("-g", "--git", action="store_true", help="mount .gitconfig")
+    parser.add_argument("-s", "--ssh", action="store_true", help="mount .ssh")
+    parser.add_argument("-t", "--tag", default="latest",
+                        help="start cs50/cli:TAG, else cs50/cli:latest", metavar="TAG")
+    parser.add_argument("-V", "--version", action="version",
+                        version="%(prog)s {}".format(__version__))
```

Comment:
Are we adding the program name in the version output? When [last we discussed this](https://github.com/cs50/style50/pull/20/files/191063f50eb199e7c7009dfc878222188066015a), you asked that `style50` only output the version number.